job-executor/src/main/java/uk/gov/moj/cpp/task/execution/JobExecutor.java

```diff
@@ -0,0 +1,153 @@
+package uk.gov.moj.cpp.task.execution;
+
+import static java.lang.Long.parseLong;
+import static java.util.UUID.randomUUID;
+import static java.util.stream.Collectors.toList;
+
+import uk.gov.justice.services.common.configuration.Value;
+import uk.gov.moj.cpp.jobstore.api.JobService;
+import uk.gov.moj.cpp.jobstore.persistence.Job;
+import uk.gov.moj.cpp.task.extension.TaskRegistry;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.ejb.Timeout;
+import javax.ejb.Timer;
+import javax.ejb.TimerConfig;
+import javax.ejb.TimerService;
+import javax.ejb.TransactionManagement;
+import javax.ejb.TransactionManagementType;
+import javax.enterprise.concurrent.ManagedExecutorService;
+import javax.inject.Inject;
+import javax.transaction.Status;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+@Startup
+@TransactionManagement(TransactionManagementType.BEAN)
+public class JobExecutor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JobExecutor.class);
```

Comment:
Inject the logger rather than using a static initializer. Just make sure LoggerProducer is on the classpath